policyengine_us/variables/household/income/person/capital_gains/capital_gains_28_percent_rate_gain.py

```diff
@@ -12,3 +12,10 @@ class capital_gains_28_percent_rate_gain(Variable):
         title="26 U.S. Code § 1(h)(4)",
         href="https://www.law.cornell.edu/uscode/text/26/1#h_4",
     )
+
+    formula = sum_of_variables(
```

Comment:
Since this is no longer a raw input, could you move this file to `variables/gov/irs/tax/federal_income/capital_gains`?